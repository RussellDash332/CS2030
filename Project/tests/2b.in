1 1
0 0.313141
0.313508 0.103753
1.204909 0.699522
2.776498 0.014224
3.876961 0.154173