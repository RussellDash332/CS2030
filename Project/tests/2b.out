0.000 1 arrives
0.000 1 serves by server 1
0.313 1 done serving by server 1
0.314 2 arrives
0.314 2 serves by server 1
0.417 2 done serving by server 1
1.205 3 arrives
1.205 3 serves by server 1
1.904 3 done serving by server 1
2.776 4 arrives
2.776 4 serves by server 1
2.791 4 done serving by server 1
3.877 5 arrives
3.877 5 serves by server 1
4.031 5 done serving by server 1
[0.000 5 0]
